endpoints.md

```diff
@@ -4,28 +4,28 @@ All endpoints that are covered in our REST API are listed here with supporting i
-|                   | GET `/projects/:id`                           | Project page                   | ✅     |
-|                   | GET `/projects/filters`                       | Project search and suggestions |        |
-|                   | GET `/projects?filter1=value1&filter2=value2` | Project search and suggestions |        |
-|                   | GET `/projects/suggestions/:id`               | Project search and suggestions |        |
-|                   | GET `/projects/:id/requests`                  | Member approval                | ✅     |
-| `/projects/`      | POST `/projects`                              | Project creation               | ✅     |
-|                   | POST `/projects/:id/requests`                 | Member approval                | ✅     |
-|                   | POST `/projects/:id/follow`                   | Project page                   | ✅     |
-|                   | POST `/projects/:id/updates`                  | Feed page                      |        |
-|                   | PUT `/projects/:id`                           | Project page                   | ✅     |
-|                   | DELETE `/projects/requests/:id`               | Member approval                | ✅     |
-| `/notifications/` | GET `/notifications/user/:id`                 | Notification system            |        |
-| `/updates/`       | GET `/updates/user/:id`                       | Feed page                      |        |
-| `/ratings/`       | POST `/ratings/`                              | Ratings and reviews            |        |
+| Prefix            | Endpoint                                                       | Frontend Component/Page        | Status |
+| ----------------- | -------------------------------------------------------------- | ------------------------------ | ------ |
+| `/auth/`          | POST `/auth/signin`                                            | Sign in                        | ✅     |
+|                   | POST `/auth/signup`                                            | Sign up                        | ✅     |
+| `/users/`         | GET `/users/:id`                                               | Profile page                   | ✅     |
+|                   | POST `/users/:id`                                              | Sign up                        |        |
+|                   | PUT `/users/:id`                                               | Profile page                   |        |
+| `/projects/`      | GET `/projects`                                                | Project search and suggestions | ✅     |
+|                   | GET `/projects/:id`                                            | Project page                   | ✅     |
+|                   | GET `/projects/filters`                                        | Project search and suggestions | ✅     |
+|                   | GET `/projects?skill[]=value1&tag[]=value2&searchQuery=value3` | Project search and suggestions | ✅     |
```

Comment:
These are all queries so you could have went with simply `search` instead of `searchQuery`.